File: codesamples/sample.go
func RemindUserWorkflow(ctx workflow.Context, userID string, intervals []int) error {
  // Send reminder emails, e.g. after 1, 7, and 30 days
  for _, interval := range intervals {
    _ = workflow.Sleep(ctx, days(interval)) // Sleep for days!
    _ = workflow.ExecuteActivity(ctx, SendEmail, userID).Get(ctx, nil) 
    // Activities have timeouts, and will be retried by default!
  }
  // ...
}

File: codesamples/sample.js
async function remindUserWorkflow(userId: string, intervals: number[]) {
  // Send reminder emails, e.g. after 1, 7, and 30 days
  for (const interval of intervals) {
    await sleep(`${interval} days`); // Sleep for days!
    await activities.sendEmail(interval, userId); // Activities retried by default!
  }
  // Easily cancelled when user unsubscribes
}
